import-dashboards.py

```diff
@@ -276,7 +276,11 @@ def add_panels():
         for file in files_list:
             with zipfile.ZipFile(os.path.join(GRAFANA_PLUGINS_DIR, file), 'r') as zip_ref:
                 print '    * Unzip %r' % (file,)
-                zip_ref.extractall(GRAFANA_PLUGINS_DIR)
+                for info in zip_ref.infolist():
+                    extracted_path = zip_ref.extract(info, GRAFANA_PLUGINS_DIR)
+                    unix_attributes = info.external_attr >> 16
```

Comment:
Please confirm: this is the right direction of shift? What permissions we should expect?

`-rw-r–r–` == `644` == `0110100100`
```
>>> 0b0110100100 >> 16
0
```

Reply:
Confirmed. We are removing extra zeros at the end that are generated by ZipInfo library. 
I have just tested the script with a zip archive of files with different permissions.

![Screenshot_20200928_140118](https://user-images.githubusercontent.com/25184101/94425337-72ef3900-0194-11eb-98ad-4e40dd0ed102.png)

![Screenshot_20200928_140159](https://user-images.githubusercontent.com/25184101/94425345-771b5680-0194-11eb-870b-57b3fa9e8a3e.png)